EN/Images/transcripts and raw translations/saber.txt

```diff
@@ -1,21 +1,21 @@
 セイバー
 Saber
 交わらぬはずの並行世界に召喚された謎を追う「騎士王」
-The Knight King who pursues the mystery of why she has been summoned to a parallel world that she is never meant to intersect with
+The King of Knights who pursues the mystery of why she had been summoned to a parallel world she was never meant to intersect with
```

Comment:
It was deliberate!!

Reply:
Wh-What does this meaaaaaaaannnnn

Reply:
Just the "King of Knights > Knight King" thing